SPEC.md

```diff
@@ -2460,6 +2604,8 @@ Would produce the command `python script.py --bams=/path/to/1.bam,/path/to/2.bam
 
 ##### Array serialization using write_lines()
 
+:pig2: [Cromwell supported](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
```

Comment:
@geoffjentry How do you feel about specifically calling out items in the WDL spec that Cromwell does not support? 

🐖  Cromwell not supported ❌  
or
🐖  coming soon in Cromwell